Sources/AsyncHTTPClient/Utils.swift

```diff
@@ -46,26 +46,6 @@ public final class HTTPClientCopyingDelegate: HTTPClientResponseDelegate {
     }
 }
 
-extension ClientBootstrap {
```

Comment:
what are the reasons for moving this out of `Utils`?

Reply:
The function creates a NIOClientTCPBootstrap now. It isn't something specific to ClientBootstrap. I could move the functions back to Utils, but it would make more sense as an extension of NIOClientTCPBootstrap

Reply:
Sure, sounds good, I just want to make sure that client pool file is not too big and filled with bunch of stuff

Reply:
done